### ISSUE TITLE
Connection Time Out Error While Connecting to Velociraptor Server

### ISSUE THREAD

**Issue by akshay91999** [weslambert/SOARLab#20]
I just installed velociraptor server on my manager node(ubuntu 20.04) and using the velociraptor_client_repacked file trying to connect to the server from client which is a sensor node(ubuntu 20.04) .
I already added the sensor node ip in the so-firewall velociraptor hostgroups and my ufw firewall is disabled in manager,still my velociraptor client is not connecting to the server,

	after running,
		 ./velcoiraptor_client_repacked client -v 
	it is showing an connection timeout error,
![Screenshot from 2023-06-01 16-41-23](https://github.com/weslambert/DinoSOARLab/assets/74290346/0914f342-dd6b-456e-8f29-fe108a4cec9f)
y its showing like this ,please help if some solution is there?

**Comment by weslambert** [comment]
Have you tried manually connecting to the server on port 8000 to make sure it is reachable?

**Comment by akshay91999** [comment]
Yes i tried on my client browser to connect to the server using https://<serverip>:8000  but it is also not reachable

**Comment by akshay91999** [comment]
Yes i tried  to connect to server 8000 port  in client browser by https://<serverip>:8000 ,but it is also unreachable

**Comment by akshay91999** [comment]
Yes i tried to connect to server 8000 port in client browser by https://serverip:8000 ,but it is also unreachable

**Comment by akshay91999** [comment]
![Screenshot from 2023-06-05 10-04-48](https://github.com/weslambert/DinoSOARLab/assets/74290346/db2be1cb-9136-47aa-826d-9ad751691a9b)
